.gitignore

```diff
@@ -3,7 +3,7 @@ syntax: glob
 ### VisualStudio ###
 
 # Tool Runtime Dir
-/[Tt]ools/
+[Tt]ools/
```

Comment:
In the future I think we should correct this in CoreFx because their version is overly restrictive.  The pattern they have there would mean nobody could have a Tools or tools folder in the entire repo -- an unreasonable (and probably unintentional) restriction.